docs/core/toc.yml

```
@@ -2,6 +2,8 @@
   href: index.md
 - name: About .NET Core
   href: about.md
+- name: Install .NET Core
```

Given that now our TOC is specific to a guide, we don't need to add .NET Core to every entry

```suggestion
- name: Install
```

---

Should I change "About .NET Core" too?